test/jdk/java/io/BufferedInputStream/LargeCopyWithMark.java

```diff
@@ -26,7 +26,7 @@
  * @requires (sun.arch.data.model == "64"  & os.maxMemory > 4g)
  * @summary BufferedInputStream calculates negative array size with large
  *          streams and mark
- * @run main/othervm -Xmx4G LargeCopyWithMark
+ * @run main/othervm -Xmx4G -Xlog:gc,gc+heap,gc+ergo+heap -XX:+CrashOnOutOfMemoryError -XX:+IgnoreUnrecognizedVMOptions -XX:+G1ExitOnExpansionFailure LargeCopyWithMark
```

Comment:
Looks okay but I assume you can split this line to avoid having a 170+ line in this file, this helps with future side-by-side review.